templates/Caillou2023Light.vue

```diff
@@ -115,3 +115,15 @@ const social = computed(() =>
     </section>
   </div>
 </template>
+
+<style scoped>
+@import url("https://fonts.googleapis.com/css2?family=Kaisei+Tokumin&display=swap");
+
+@import "@/assets/styles/index.css";
+
+@layer components {
+  .font-display {
+    font-family: "Kaisei Tokumin";
+  }
+}
```

Comment:
Je suis pas convaincu de l'utilité de `@layer components` dans ce cas et même de l'import de `index.css`

Est-ce qu'on peut juste faire ça ?
```vue
<style scoped>
@import url("https://fonts.googleapis.com/css2?family=Kaisei+Tokumin&display=swap");

.font-display {
  font-family: "Kaisei Tokumin";
}
</style>
```

---

Le doc de tailwind sur *layer* ici : https://tailwindcss.com/docs/functions-and-directives#layer

Pour moi ça sert à créer des classes réutilisables globalement avec le fonctionnement de *tailwind* (les modifiers, `lg:`, `hover:`, etc).